app/src/main/java/com/github/se/eventradar/model/event/EventCategory.kt

```diff
@@ -1,12 +1,28 @@
 package com.github.se.eventradar.model.event
 
+import android.content.Context
+import com.github.se.eventradar.R
+
 enum class EventCategory(val displayName: String) {
   MUSIC("Music"),
   SPORTS("Sports"),
   CONFERENCE("Conference"),
   EXHIBITION("Exhibition"),
   COMMUNITY("Community"),
   SOCIAL("Social"),
+  PARTY("Party");
+
+  fun toString(context: Context): String {
```

Comment:
this is already implemented by [Kotlin](https://stackoverflow.com/questions/28548015/how-do-i-create-an-enum-from-a-string-in-kotlin)

Reply:
I know there already exist a function from kotlin that will convert the enum value to string, but the idea (maybe I think touo far) is to map each type of the enum to a Ressource String, which as I understood is of use to easily translate the app and its content. Since I used string ressource for subtitle such as "description", "location" I wanted to stay consistent and have string ressource to display the event Category as well